_data/glossary/resource-quota.yaml

```diff
@@ -0,0 +1,11 @@
+id: resource-quota
+name: Resource Quotas
+full-link: /docs/concepts/policy/resource-quotas/
+tags:
+- fundamental
+- operation
+- architecture
+short-description: >
+  Provides constraints that limit aggregate resource consumption per {% glossary_tooltip term_id="namespace" %}.
```

Comment:
s/Provides/Provide
or else make entry singular (Resource Quota)

Reply:
Second to making it singular.